### DISCUSSION TITLE
Proposal: Module backwards compatibility V3

### DISCUSSION THREAD

**Discussion by rogpeppe** [cue-lang/cue#2943] [Proposal]
Proposal https://github.com/cue-lang/cue/discussions/2939 relies on MVS, which requires a notion of backwards compatibility. This is discussed in this sub-proposal:
https://github.com/cue-lang/proposal/blob/main/designs/modules.v3/2943-modules-compat.md

**Comment by myitcv** [comment]
Based on the excellent feedback received from CUE users to date, and updates shared on the various [modules update calls](https://github.com/cue-lang/cue/discussions/categories/community-calls?discussions_q=is%3Aopen+category%3A%22Community+Calls%22+label%3Amodules), the consensus appears to be that the [parent proposal](https://github.com/cue-lang/cue/discussions/2939), this sub-proposal and its siblings should be accepted.

As discussed on the [2024-05-02 modules update call](https://github.com/cue-lang/cue/discussions/3068), we are therefore moving the parent proposal and its sub-proposals to a status of **"likely accept"**. 

This marks the final period for comments that might change the recognition of consensus. We are aiming to conclude this decision at the [next Modules Feedback session (2024/05/16)](https://github.com/cue-lang/cue/discussions/3119). If there is no change in that consensus, we will accept the parent proposal and its sub-proposals on that date.

**Comment by myitcv** [comment]
On the [2024-05-16 Modules update call](https://github.com/cue-lang/cue/discussions/3119) we [accepted](https://docs.google.com/presentation/d/17VtQs3QitwXFTBI5BT69h6MzPvEEOlqJtQ7XYCExl7M/edit#slide=id.g2bbab775c99_0_34) [v3 of the modules proposal and its sub-proposals](https://github.com/cue-lang/cue/discussions/2939). Closing this discussion to recognise that decision.
